src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -17,6 +17,15 @@
         Gives X amount in the past NOT including the current second.
         "-t 1m" with now:1000 (start:940, end:999)
 
+
+    One Argument and an option
+    --------------------------
+
+    Timerange rounded
+        Gives X amount in the past NOT including the current second, rounded.
+        "-t 2m --round" with now:12:34:15 (start:12:32:00, end:12:33:59)
```

Comment:
I find that hard to understand from this description what actually happenes. I had to look into the implementation. Maybe it could at least be clarified what rounded means in this context.

Reply:
Fixed.